control/check_install.sh

```diff
@@ -45,13 +46,11 @@ fi
 
 if [ ! -f "${FAB_CONFIG}" ]
 then
-    for OLD_FAB_DIR in "${COMMCARE_CLOUD_REPO}/../commcare-hq/deployment/commcare-hq-deploy" \
-                       "${COMMCARE_CLOUD_REPO}/../commcare-hq-deploy" \
-                       "${COMMCARE_CLOUD_REPO}/fab"
+    for OLD_FAB_CONFIG_DIR in "${COMMCARE_CLOUD_REPO}/src/commcare_cloud/fab/config.py"
```

Comment:
This is a file path, not a directory, which I think will not work correctly below. Should this `for`-loop be eliminated?

Reply:
Oops, sorry this is a copy-paste error. I'll fix this

Reply:
Okay updated and tested this directly. Thanks for catching

Reply:
Addressing the for-loop suggestion as well

Reply:
Addressed